onyo/cli/tree.py

```diff
@@ -20,8 +20,8 @@
 
 
 def tree(args: argparse.Namespace) -> None:
-    """
-    List the assets and directories of **DIRECTORY**\s in a tree-like format.
+    r"""
+    List the assets and directories of **DIRECTORY**\ s in a tree-like format.
```

Comment:
Again, I know that you are just reformatting, without necessarily reading these all.
But I think in another place, instead of "in a tree-like format" we say "in the tree format"?

Reply:
I like the phrasing "in a tree-like format." For we are formatting it like a tree, and like `tree`.